engine/crates/engine/src/schema.rs

```diff
@@ -466,10 +474,12 @@ impl Schema {
         request: impl Into<Request> + Send,
         session_data: Arc<Data>,
     ) -> impl Stream<Item = StreamingPayload> + Send + Unpin {
+        let start = web_time::Instant::now();
         let schema = self.clone();
-        let request = request.into();
+        let request: Request = request.into();
         let extensions = self.create_extensions(session_data.clone());
         let gql_span = GqlRequestSpan::new().into_span();
+        let normalized_query = operation_normalizer::normalize(request.query(), request.operation_name()).ok();
```

Comment:
Not much we can do to avoid it I supppose but wonder what impact adding this is going to have on the wasm size...

Reply:
we do it already in the worker anyway already, so won't change anything

Reply:
We do it in the gateway currently, but this is going to run in the executor, no?  Hence wondering

Reply:
hum good point, you're right